app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,33 @@
-<h2>Log in</h2>
+</html> 
+<head>
```

Comment:
Put this in `application.html.erb` if you want it to be applied throughout the whole website ;)